third_party/xla_client/xrt_computation_client.cc

```diff
@@ -328,13 +329,18 @@ std::vector<ComputationClient::DataPtr>
 XrtComputationClient::TransferToServerInternal(
     absl::Span<const TensorSource> tensors) {
   metrics::TimedSection timed(TransferToServerMetric());
+  tensorflow::profiler::TraceMe activity(
+    "TransferToServer", tensorflow::profiler::TraceMeLevel::kInfo);
```

Comment:
consider putting some of these strings into constants, especially if used multiple times

Reply:
None of these are used more than once as they're simply the method's name (but yes, there was a typo in this instance, which I've fixed). The recommended use is to keep as string literal when using `TraceMe` annotations. Just as a single example: https://github.com/tensorflow/tensorflow/blob/684632e672f1aa0b0554f482b3459e3cd32216bd/tensorflow/compiler/xrt/kernels/tpu_execute_op.cc#L64